cardano-db-sync/test/Cardano/DbSync/ApiTest.hs

```diff
@@ -22,46 +23,50 @@ prop_extractInsertOptions :: Property
 prop_extractInsertOptions = property $ do
   cfg <- forAll Gen.syncPreConfig
 
-  let insertOpts = pcInsertConfig cfg
-  coverInsertCfg insertOpts
+  let insertCfg = pcInsertConfig cfg
+  coverInsertCfg insertCfg
 
-  case insertOpts of
-    FullInsertOptions ->
+  case insertCfg of
+    SyncInsertConfig (Just "full") _ ->
```

Comment:
These would be nicer to use as ADTs (rather than strings), as I mentioned in my other comment